src/containers/CardDeck.js

```diff
@@ -0,0 +1,337 @@
+									"marks": [
+										{
+											"type": "strong"
+										}
+									],
+									"text": "Hype! "
+								}
+							]
+						}
+					]
+				},
+				"selection": {
+					"type": "text",
+					"anchor": 0,
+					"head": 0
+				}
+			},
+			{
+				"doc": {
+					"type": "doc",
+					"content": [
+						{
+							"type": "heading",
+							"attrs": {
+								"hidden": false,
+								"level": 1
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Thank You"
+								}
+							]
+						},
+						{
+							"type": "paragraph",
+							"attrs": {
+								"hidden": false
+							},
+							"content": [
+								{
+									"type": "text",
+									"marks": [
+										{
+											"type": "code_inline"
+										}
+									],
+									"text": "print(\"Hope you liked this top secret presentation\")"
+								}
+							]
+						},
+						{
+							"type": "paragraph",
+							"attrs": {
+								"hidden": false
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Hope you liked this "
+								},
+								{
+									"type": "text",
+									"marks": [
+										{
+											"type": "strong"
+										}
+									],
+									"text": "top secret presentation"
+								}
+							]
+						}
+					]
+				},
+				"selection": {
+					"type": "text",
+					"anchor": 0,
+					"head": 0
+				}
+			}
+		]
+		const backJSON = [
+			{
+				"doc": {
+					"type": "doc",
+					"content": [
+						{
+							"type": "heading",
+							"attrs": {
+								"hidden": false,
+								"level": 1
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Back of Card 1"
+								}
+							]
+						}
+					]
+				},
+				"selection": {
+					"type": "text",
+					"anchor": 0,
+					"head": 0
+				}
+			},
+			{
+				"doc": {
+					"type": "doc",
+					"content": [
+						{
+							"type": "heading",
+							"attrs": {
+								"hidden": false,
+								"level": 1
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Back of Card 2"
+								}
+							]
+						},
+						{
+							"type": "paragraph",
+							"attrs": {
+								"hidden": false
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Flippity flip "
+								},
+								{
+									"type": "text",
+									"marks": [
+										{
+											"type": "strong"
+										}
+									],
+									"text": "flip! "
+								}
+							]
+						}
+					]
+				},
+				"selection": {
+					"type": "text",
+					"anchor": 0,
+					"head": 0
+				}
+			}
+		];
+		const content = [
+			{"index": 0, "front": frontJSON[0]},
+			{"index": 1, "front": frontJSON[1], "back": backJSON[0]},
+			{"index": 2, "front": frontJSON[2], "back": backJSON[1]},
+			{"index": 3, "front": frontJSON[3]}
+		]
+		this.setState({content: content})
+	}
+	
+	render = () => {
+		if (this.state.content == null){
+			return (
+				<Container>
+					<Card/>
```

Comment:
tbh I think it's better to just show an empty page, no default blank card at all

Reply:
yeah my bad, that was a remnant from testing. will remove